text/0000-macros-1.1.md

```diff
@@ -492,3 +520,9 @@ pub struct Foo {
   we can layer on features backwards-compatibly to get to macros 2.0. Right now,
   though, the delta between what this RFC proposes and where we'd like to is
   very small, and can get get it down to actually zero?
+
+* Eventually macro crates will want to be loaded both at compile time and
+  runtime, and this means that Cargo will need to understand to compile these
+  crates twice, once as `rustc-macro` and once as an rlib. Does Cargo have
```

Comment:
Perhaps, yeah, but the compiler would have to run itself twice basically anyway I think because:

* What's actually compiled changes
* LLVM can only generate one target at a time
* The dependencies for each target may be different

etc. Some of this definitely seems like Cargo's domain?